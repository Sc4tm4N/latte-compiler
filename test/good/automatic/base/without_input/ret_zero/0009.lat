// print string some equations (compared to C)
int main() {
  string s1 = "FoRtNit3 " + "to" + " fcale nie ZacnaGier" + "ka" + "!11!",
    s2 = s1 + "\n - ZGADZAM SIE W 1)0!!!!1 procentach!";
  printString(s1);
  printString(s2);
  printString("a" + "b" + "c" + "d" + "\"" + "f" + "\"");
  return 0;
}