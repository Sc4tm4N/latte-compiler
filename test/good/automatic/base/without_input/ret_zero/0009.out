FoRtNit3 to fcale nie ZacnaGierka!11!
FoRtNit3 to fcale nie ZacnaGierka!11!
 - ZGADZAM SIE W 1)0!!!!1 procentach!
abcd"f"
